sky/tools/sky_utils.py

```diff
@@ -37,7 +39,11 @@ def assert_valid_codesign_config(framework_dir, zip_contents, entitlements, with
     log_error('ERROR: duplicate value(s) found in without_entitlements.txt')
     sys.exit(os.EX_DATAERR)
 
-  if _contains_duplicates(entitlements + without_entitlements):
+  if _contains_duplicates(unsigned_binaries):
+    log_error('ERROR: duplicate value(s) found in unsigned_binaries.txt')
```

Comment:
Should we update `_contains_duplicates()` to actually log the duplicates?

Reply:
Yes! But, check line 46 below. Default diff algorithm is unfortunate.

Reply:
what about at this point, when we're definitely gonna exit with an error anyway, we call ANOTHER function, `def print_duplicates(list)`? I'm just thinking if someone hits this error and doesn't know what it means, it might be really painful figuring out what they forgot to do.

Reply:
Sorry missed your comment; followup here: https://github.com/flutter/engine/pull/54987